Cisco ISR - Updating modeling rules (#39595)
demisto/content: Packs/CiscoISR/README.md

> # Cisco ISR

Reviewer: nvm, pack is only for XSIAM

More information can be found [here](https://www.cisco.com/c/en/us/td/docs/routers/sdwan/command/iosxe/qualified-cli-command-reference-guide/m-logging-commands.pdf)

### Syslog Time Parsing Support

> Syslog timestamp parsing by default support UTC timezone (offset "+00:00").  

Reviewer 2: ```suggestion
By default, syslog timestamp parsing supports a UTC timezone (offset "+00:00").  
```

> To support other timezones you will need to modify Parsing Rules in **Data Management** &rarr; **Data Model Rules** and select **User Defined** tab, change the UTC offset in _parse_timestamp_ functions to the relevant timezone.  

Reviewer 2: ```suggestion
To support other timezones you will need to modify Parsing Rules in **Data Management** &rarr; **Data Model Rules** and select the **User Defined** tab, and change the UTC offset in _parse_timestamp_ functions to the relevant timezone.  
```

And wait for system confirmation.

Timestamp ingestion is currently supported for the following formats in UTC time zone;

> * mmm d HH:MM:SS.sss - E.g. Jan 9 09:00:00.321 
> * mmm dd HH:MM:SS.sss - E.g. Jan 01 10:00:00.123
> * mmm  d YYYY HH:MM:SS.sss - E.g. Feb  3 2025 22:04:01.776
> * mmm dd YYYY HH:MM:SS.sss - E.g. Mar 01 2021 10:00:00.123

Reviewer 2: ```suggestion
* mmm d HH:MM:SS.sss - E.g., Jan 9 09:00:00.321 
* mmm dd HH:MM:SS.sss - E.g., Jan 01 10:00:00.123
* mmm  d YYYY HH:MM:SS.sss - E.g., Feb  3 2025 22:04:01.776
* mmm dd YYYY HH:MM:SS.sss - E.g., Mar 01 2021 10:00:00.123
```
